indexing-service/src/main/java/org/apache/druid/indexing/common/task/batch/parallel/GeneratedPartitionsReport.java

```diff
@@ -91,6 +108,7 @@ public String toString()
         "taskId='" + taskId + '\'' +
         ", partitionStats=" + partitionStats +
         ", taskReport=" + taskReport +
+           ", segmentsRead=" + segmentsRead +
```

Comment:
nit: can make the indentation more consistent

Reply:
```suggestion
        ", segmentsRead=" + segmentsRead +
```